spec/factories/book.rb

```diff
@@ -1,10 +1,10 @@
 FactoryBot.define do
     factory :book do
-      genre 'Novela'
-      author 'Anthony Burgess'
-      image 'naranja_mecanica.jpg'
-      title 'La naranja mecanica'
-      publisher 'Editorial Sarlanga'
-      year '1970'
+      genre Faker::Book.genre
```

Comment:
Put the fakers between brackets so as to generate different data each time. If not all will have the same data

Reply:
I really did not know, thanks!!